EngineBay.DemoModule/Queries/QueryTodoItem.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace EngineBay.DemoModule
+{
+    using System.Globalization;
+    using System.Linq.Expressions;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using EngineBay.Core;
+    using LinqKit;
+    using Microsoft.EntityFrameworkCore;
+
+    public class QueryTodoItem : PaginatedQuery<TodoItem>, IQueryHandler<QueryTodoItemRequest, PaginatedDto<TodoItemDto>>
```

Comment:
Why don't we pass PaginationParameters directly? Do we anticipate this request growiung?

Reply:
Perhaps the filtering, though I'm looking into that now